learn/parse_http.go

```diff
@@ -358,19 +358,40 @@ func parseBody(contentType string, bodyStream io.Reader, statusCode int) (*pb.Da
 			return nil, errors.Wrapf(err, "could not parse YAML body")
 		}
 		pbContentType = pb.HTTPBody_YAML
+	case "text/html":
+		body, err := limitedBufferBody(bodyStream)
+		if err != nil {
+			return nil, err
+		}
+		// This is a fairly cheesy way of handling the body, is there a better one?
+		bodyData = parseElem(string(body), spec_util.INTERPRET_STRINGS)
```

Comment:
Seems not worth trying to interpret strings: if it claims to be HTML, it will probably have tags in it, and is therefore unlikely to be successfully parsed as an int, float, or bool.

Reply:
Fixed in https://github.com/akitasoftware/akita-cli/pull/111/commits/b88090ba52de7c92877db4ea862c3fbbd69c7f1b